packages/serverpod_serialization/lib/src/serialization.dart

```diff
@@ -78,7 +91,9 @@ abstract class SerializationManager {
       if (data == null) return null as T;
       return UuidValueJsonExtension.fromJson(data) as T;
     }
-    throw FormatException('No deserialization found for type $t');
+    throw SerializationTypeNotFoundException(
+      'No deserialization found for type $t',
```

Comment:
Suggestion: Store the type in `SerializationTypeNotFoundException`.

This allows the developer that catches the exception to inspect the type.